packages/atomic/src/components/atomic-search-box/atomic-search-box.pcss

```diff
@@ -1,6 +1,10 @@
 @import "tailwindcss/base";
 @import "tailwindcss/utilities";
 
+atomic-search-box {
```

Comment:
why is this necessary? Could we keep a tailwind class `z-50` on the suggestions?